### DISCUSSION TITLE
W3C Web of Things Dialogue at TPAC

### DISCUSSION THREAD

**Discussion by egekorkan** [home-assistant/architecture#961] [General]
Hi Home Assistant Community!

I am one of the co-chairs of the W3C Web of Things Community Group. Every year the W3C meets for the TPAC event and this year, we are proposing a session for having an open dialogue with the Home Assistant Community. 

The session proposal is online at https://github.com/w3c/tpac2023-breakouts/issues/50.  Among other applications, WoT can be used for reducing the friction of adding integrations to Home Assistant. 

We would like to invite members of the Home Assistant community to join us to discuss how the two communities can collaborate in the future.

Some links for background:
- [WoT Homepage](https://www.w3.org/WoT/)
- [Explainer Page](https://www.w3.org/WoT/documentation/)
- [Thing Description Standard](https://www.w3.org/TR/2020/REC-wot-thing-description-20200409/)

**Comment by parautenbach** [comment]
This should probably rather be posted under the social section of the forum: https://community.home-assistant.io/c/social/46.

**Comment by egekorkan** [comment]
Sorry about that, I have posted it at https://community.home-assistant.io/t/w3c-web-of-things-dialogue-at-w3c-tpac/609764
